packages/expect/__typetests__/expect.test.ts

```diff
@@ -16,7 +16,8 @@ import {
 } from 'expect';
 import type * as jestMatcherUtils from 'jest-matcher-utils';
 
-type M = Matchers<void>;
+type M = Matchers<void, unknown>;
+type N = Matchers<void>;
```

Comment:
what is this `N` for? 🤔

Reply:
Just three test – takes two type args, but allows passing juts one and (below) errors if type args are missing.

Some time ago I added second arg without default. That was a breaking change and a user raised an issue. This test was added back in these days, but became unnecessary then `@jest/expect` package was introduced. Now it is needed again.

One day this will be wrapped with nice `test("required type arguments", () => { /...`. Good enough for now (;